src/components/input/model.ts

```diff
@@ -0,0 +1,9 @@
+export interface Input {
+    width?: string;
+}
+
+export interface InputProps extends Input {
+    onChange?(): void;
```

Comment:
Without event arg I am not able to target input value on change. This fn should have optional event arg.